insitupy/io/metadata.py

```diff
@@ -1,43 +1,51 @@
 import logging
-from typing import List
+from pathlib import Path
+from typing import List, Tuple, Optional, Union
 
 import pandas as pd
 
 from .dates import DateManager
 from .locations import LocationManager
 from .strings import StringManager
 from .yaml_codes import YamlCodes
-from ..profiles.metadata import ProfileMetaData
-from ..variables import ExtendableVariables
+from insitupy.variables import (
+    ExtendableVariables,
+    base_metadata_variables_yaml, base_primary_variables_yaml
+)
+from insitupy.profiles.metadata import ProfileMetaData
 
 LOG = logging.getLogger(__name__)
 
 
 class MetaDataParser:
     """
-    Base class for parsing metadata
+    Base class for parsing metadata in the header and data column names
     """
+    DEFAULT_METADATA_VARIABLE_FILES = [base_metadata_variables_yaml]
+    DEFAULT_PRIMARY_VARIABLE_FILES = [base_primary_variables_yaml]
```

Comment:
Moved this to here since it felt more natural to have this handled in the `MetaDataParser` class as both information pieces are found within the header